packages/flutter_tools/templates/create/.atom/launches.tmpl/main.yaml.tmpl

```diff
@@ -0,0 +1,10 @@
+# Flutter launch configuration for lib/main.dart.
+type: flutter
+path: lib/main.dart
+
+flutter:
+  checked: true
+  debug: true
```

Comment:
We should add some more comment explaining what these various options control.  Doesn't have to be in this patch.